File: src/net/eduard/api/server/SoulSystem.java
package net.eduard.api.server;

import net.eduard.api.lib.modules.FakePlayer;

public interface SoulSystem {
	
	public void addSouls(FakePlayer player,double amount);
	
	public void removeSouls(FakePlayer player,double amount);
	
	public double getSouls(FakePlayer player);
	
	public void setSouls(FakePlayer player,double amount);

}

File: src/main/java/net/eduard/api/server/SoulSystem.kt
package net.eduard.api.server

import net.eduard.api.lib.modules.FakePlayer


interface SoulSystem {

    fun addSouls(player: FakePlayer, amount: Double)
    fun removeSouls(player: FakePlayer, amount: Double)
    fun getSouls(player: FakePlayer): Double
    fun setSouls(player: FakePlayer, amount: Double)

}
